index.rst

```diff
@@ -326,33 +326,167 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
+The data is organized by topics following the `T&S XML schema`_ and recorded in InfluxDB, a time series database optimized for queries constrained by time on large datasets.
+
+The EFD is managed by `Sasquatch`_, an RSP service that includes tools like Chronograf for data exploration and visualization and Kapacitor for alerting on the EFD data
+(see also :ref:`Metric databases in Sasquatch <metric-databases>`).
+
+.. _the-butler-registy:
+
+The Butler registry
+"""""""""""""""""""
+
+Rapid Analysis will use the Science Pipelines infrastructure for SFP.
+That infrastructure provides a Butler registry, a database for tracking processing input and output datasets.
+
+The current plan is to have a Butler repository per instrument at the Summit.
+
+Once the exposures are ingested into the Butler "raw" collection, IDs are assigned, and the registry becomes the primary source for obtaining the list of observations.
+
+The Butler registry schema is described in `DMTN-073`_.
+It is designed to support visits composed of multiple exposures.
+If the visit is composed of a single exposure, then the visit table has entries that are essentially duplicates of the science exposures in the exposure table.
+
+However, the registry database contains only a small subset of the raw image metadata (the information about the conditions under which an image was taken summarized in the image headers).
+
+Analysis outputs are not in the registry database; they are typically linked to the visit table in the registry and stored as files in the Butler datastore.
+An example of an SFP task that produces such ouput is the `ConsolidateVisitSummaryTask`_ task that persists per-detector summary statistics computed for calibrated exposures into a per-visit dataset in Parquet files.
+
+Expanding the Butler registry database to include the required raw image metadata and analysis outputs is outside the plans.
+Hence the need for a :ref:`Visit database <visit-database>` (see also `DMTN-227 The Consolidated Database of Image Metadata`_).
+
+.. _visit-database:
+
+The Visit database
+""""""""""""""""""
+
+The initial content for the Visit database is described in the database `use cases`_ confluence page and summarized in this section.
+We also discuss implementation details for creating and updating the Visit database tables.
+
+Similarly to the Butler registry, a PostgreSQL database is the natural choice for the Visit database implementation.
+The Visit database should be replicated at the USDF, and the Summit instance should keep data for at least 30 days.
+
+**Visit and detector tables**
+
+The image headers and the Rapid Analysis outputs are the primary sources of information for the visit and detector tables.
+
+Rapid Analysis will have access to the headers during processing and can consolidate the raw image metadata into per-detector, and per-visit datasets along with the per-visit summary statistics from SFP persisted as Butler datasets.
```

Comment:
I think DM calls it "SFM" not "SFP" (M: Measurement).  If it's not hard, can we switch to SFM and provide a definition?

FYI, I don't expect that much extra metadata will be of interest, we should concentrate on the numbers calculated by the pipelines.

Reply:
We have used the Single Frame Processing terminology throughout the report, and I think we really mean that, i.e. [ISR and Image Characterization](https://pipelines.lsst.io/getting-started/singleframe.html) as opposed to the [SingleFrameMeasurementTask](https://pipelines.lsst.io/modules/lsst.meas.base/tasks/lsst.meas.base.sfm.SingleFrameMeasurementTask.html).

Reply:
I don't think so.  The photometry and PSF estimation comes from SingleFrameMeasurement.

The name isn't critical; let's just make sure that we define the acronym and agree what we mean!